layout/net461/net461.depproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <NuGetTargetMoniker>.NETFramework,Version=v4.6.1</NuGetTargetMoniker>
```

Comment:
I decided that because of the netstandard20 mapping goes to net461. We can always decide to update .net core to support different versions as needed.